contracts/drafts/issuance/Issuance.sol

```diff
@@ -113,6 +116,34 @@ contract Issuance is Ownable, StateMachine {
         emit InvestmentAdded(msg.sender, _amount);
     }
 
+    function withdraw() external nonReentrant {
+        require(currentState == "DISTRIBUTING", "Cannot withdraw now.");
```

Comment:
You probably don't need a DISTRIBUTING state and can move directly from OPEN to LIVE, allowing investors to withdraw if we are LIVE.